packages/docs-app/src/examples/select-examples/selectExample.tsx

```diff
@@ -106,4 +120,6 @@ export class SelectExample extends React.PureComponent<IExampleProps, ISelectExa
             this.setState(state => ({ ...state, [prop]: checked }));
         };
     }
+
+    private isItemDisabled = (film: IFilm) => this.state.disableItems && film.year < 2000;
```

Comment:
this works because no films were ever produced prior to Jan 1, 1990 :stuck_out_tongue_winking_eye: